playground/src/fly-in/fly-in-background.scss

```diff
@@ -14,7 +14,7 @@
   transition: opacity .25s ease-in-out
 }
 
-.active ~ #fly-in-background {
+.lightbox.active ~ #fly-in-background {
```

Comment:
Hm... we are using both: simple state classes (`.active`, `.open`, ...) and BEM modifiers (`.*--left`, `.*--hidden`). Shouldn't we commit and then stick to one of them?